GitHubComments.ps1

```diff
@@ -62,33 +62,37 @@ function Get-GitHubIssueComment
         [Parameter(Mandatory)]
         [int] $IssueNumber,
 
-        [string] $Since,
+        [DateTime] $Since,
 
         [string] $AccessToken,
 
         [switch] $NoStatus
     )
 
-    Write-InvocationLog -Invocation $MyInvocation
+    Write-InvocationLog
 
-    $elements = Resolve-RepositoryElements -BoundParameters $PSBoundParameters
+    $elements = Resolve-RepositoryElements
     $OwnerName = $elements.ownerName
     $RepositoryName = $elements.repositoryName
 
+    if ($Since -ne $null) {
+        $SinceFormattedTime = $Since.ToUniversalTime().ToString('o')
+    }
+
     $telemetryProperties = @{
         'OwnerName' = (Get-PiiSafeString -PlainText $OwnerName)
         'RepositoryName' = (Get-PiiSafeString -PlainText $RepositoryName)
-        'Number' =  $IssueNumber
-        'Since' = $Since
+        'IssueNumber' =  $IssueNumber
+        'Since' = $SinceFormattedTime
```

Comment:
For things like this, I don't think it matters as much what this value was.  From a telemetry perspective, we'd more possibly be interested in _if_ they provided it at all.  You'll see [other instances](https://github.com/PowerShell/PowerShellForGitHub/blob/master/GitHubIssues.ps1#L179) where we just log whether or not the user provided a value for a parameter, as opposed to the value of the parameter itself.  It's more about being able to answer the question: "how are people using this function".  Tracking the OnwerName/RepositoryName allows us to identify the breadth of the usage across GitHub, while things like this allow us to track how an individual function is getting used.

Reply:
That makes sense, updated this to be a bool